components/gitpod-protocol/src/public-api-converter.ts

```diff
@@ -427,4 +435,74 @@ export class PublicAPIConverter {
         }
         return result;
     }
+
+    toAuthProviderDescription(ap: AuthProviderInfo): AuthProviderDescription {
```

Comment:
please add tests to the corresponding spec file

Reply:
done